pages/room/[id].js

```diff
@@ -177,6 +206,51 @@ export default function Room(props) {
       console.log(e)
     }
   }
+
+  const removeOnePlayer = async (playerId) => {
+    let playerIdToRemove = playerId || userId
+    const index = room.userIds.indexOf(playerIdToRemove)
+
+    if (index > -1) {
+      let copiedRoomUserIds = [...room.userIds]
+      copiedRoomUserIds.splice(index, 1)
+      console.log(copiedRoomUserIds)
+      try {
+        // remove user from room
+        await updateRoom({
+          variables: {
+            updateRoomId: room.id,
+            updateRoomUsers: copiedRoomUserIds,
+            isShown: room.isShown,
+          },
+        }).catch((e) => {
+          throw e
+        })
+        // remove user from backend
+        deleteUser({
+          variables: {
+            deleteUserId: playerIdToRemove,
+          },
+        }).catch((e) => {
+          throw e
+        })
+      } catch (e) {
+        console.log(e)
+      }
+    }
+  }
+
+  const destroyRoom = () => {}
+
+  const leaveRoomClick = async () => {
+    if (userId === room.host) {
+      console.log(
```

Comment:
I'm assuming this is a place holder for destroyRoom.
Either remove destroyRoom or call it here and console log the reminder from the placeholder function